Cylenium.Tests/DriverTests.cs

```diff
@@ -39,6 +40,30 @@ public void Set_and_get_the_window_size()
             Assert.AreEqual(cy.WindowSize(), expectedSize);
         }
 
+        [Test]
+        [Category("Driver")]
+        public void Navigate_backward_and_forward()
+        {
+            var wait = cy.Wait();
+
+            cy.Visit("https://google.com");
+            wait.Until(_ => cy.Get("[name='q']"));
+            var first = cy.Title();
```

Comment:
Make the variable name more explicit so it makes more sense later on in the code.

```python
var first_title = cy.Title();
```

Reply:
or maybe just call it `google_title` and `ultimateqa_title`